packages/eslint-config-mavenlint/index.js

```diff
@@ -2,10 +2,7 @@ module.exports = {
   extends: 'airbnb',
   plugins: ['mavenlint', 'jasmine'],
   rules: {
-    'import/no-extraneous-dependencies': [
-      'off',
-      { devDependencies: true, optionalDependencies: false, peerDependencies: false },
-    ],
+    'import/no-extraneous-dependencies': 'error',
```

Comment:
I think we can just remove the override. AirBnB configuration might be using reasonable settings.

https://github.com/airbnb/javascript/blob/32a7bf44c2e5b3d2a5d05af46716fbe45bc09bec/packages/eslint-config-airbnb-base/rules/imports.js#L71-L94

Reply:
Good idea